spec/features/admin_claim_tasks_update_with_dqt_api_spec.rb

```diff
@@ -102,147 +101,1323 @@ def task_outcome
+            before { visit admin_claim_tasks_path(claim) }
+
+            scenario "shows identity confirmation passed" do
+              expect(task("Identity confirmation")).to have_text("Partial match")
+            end
+          end
+
+          context "admin claim tasks identity confirmation view" do
+            before { visit admin_claim_task_path(claim, :identity_confirmation) }
+
+            scenario "shows identity confirmation question" do
+              expect(page).to have_content("Did #{claim.full_name} submit the claim?")
+              expect(page).to have_link(href: admin_claim_notes_path(claim))
+            end
+          end
+
+          context "admin claim notes view" do
+            before { visit admin_claim_notes_path(claim) }
+
+            scenario "shows first name or surname not matched by an automated check" do
+              expect(notes).to include(
+                have_text(
+                  "First name or surname not matched"
+                ).and(
+                  have_text(
+                    "by an automated check on #{I18n.l(claim.notes.last.created_at)}"
+                  )
+                )
+              )
+            end
+          end
+        end
+
+        context "except matching surname" do
+          let(:data) do
+            {
+              date_of_birth: claim.date_of_birth,
+              name: "#{claim.first_name} Except",
+              national_insurance_number: claim.national_insurance_number,
+              teacher_reference_number: claim.teacher_reference_number
+            }
+          end
+
+          context "admin claim tasks view" do
+            before { visit admin_claim_tasks_path(claim) }
+
+            scenario "shows identity confirmation passed" do
+              expect(task("Identity confirmation")).to have_text("Partial match")
+            end
+          end
+
+          context "admin claim tasks identity confirmation view" do
+            before { visit admin_claim_task_path(claim, :identity_confirmation) }
+
+            scenario "shows identity confirmation question" do
+              expect(page).to have_content("Did #{claim.full_name} submit the claim?")
+              expect(page).to have_link(href: admin_claim_notes_path(claim))
+            end
+          end
+
+          context "admin claim notes view" do
+            before { visit admin_claim_notes_path(claim) }
+
+            context "admin claim notes view" do
+              before { visit admin_claim_notes_path(claim) }
+
+              scenario "shows first name or surname not matched by an automated check" do
+                expect(notes).to include(
+                  have_text(
+                    "First name or surname not matched"
+                  ).and(
+                    have_text(
+                      "by an automated check on #{I18n.l(claim.notes.last.created_at)}"
+                    )
+                  )
+                )
+              end
+            end
```

Comment:
@nomasprime Why the double nesting? This to me looks to be an error only a single context. Am I missing something or is another state missing that caused you to present this as a context 6 levels deep?

Reply:
Apologies I should have been clearer on this Level 6 nesting question. Level 5 starts on L316 and runs through L335.
Level 6 starts on L319 and runs through L333 (as I originally highlighted).

My understanding is normally contexts are in pairs, and this looks to be an alignment issue as why would we need the following nesting?
```
          context "admin claim notes view" do
            before { visit admin_claim_notes_path(claim) }

            context "admin claim notes view" do
              before { visit admin_claim_notes_path(claim) }
```

Reply:
Updated.